### PR TITLE
Doc cleanup and examples

### LINKED ISSUE #13: Documentation: "Example Workflows"

### LINKED ISSUE #159: Write post on xclim's progress
### Description

Write a couple of paragraphs on xclim, usage and performance.

### PR DESCRIPTION
**Pull request by Zeitsperre**
* **Please check if the PR fulfills these requirements**
- [x] This PR addresses an already opened issue (for bug fixes / features)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] HISTORY.rst has been updated
- [ ] bumpversion (minor / major / patch) has been called
- [ ] Tags have been pushed

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, etc.)
Changed some wording in the docs for grammar and consistency with development practices

* **Does this PR introduce a breaking change?** (Has there been an API change?)
No.

* **Other information**:
fixes #159 
fixes #13 

* When merging, please put `fixes #{issue number}` in your comment to auto-close the issue that your PR addresses. Thanks!

### CODE REVIEW

**Comment by Zeitsperre** [comment]
For reference, the issue with integrating Jupyter Notebooks into the docs here is due to a few factors/bugs:
- Pandoc is needed to parse the cells and create sphinx text out of them. If the HOME is not set in the environment (ie: the TOXENV), the build will fail. This is well documented [here](https://github.com/bebraw/pypandoc/pull/104)
- TODO: Ipython

**Comment by Zeitsperre** [comment]
@huard Thanks for noting that. I almost forgot!

I'll ask @tlogan2000 to re-run the cells as the outputs are not 100% correct at the moment. Some of the examples were based on files on his desktop or accessible via mounts he configured. Once the cells are updated, I'll send another review request.

**Comment by huard** [comment]
Ideally the examples would be self-contained, so that anyone can run them and get the same answer.

**Comment by Zeitsperre** [comment]
Good point. 

I'll see if I can simplify the MF-dataset example. It's the only one that would be complicated without access to the database. Might solve more than one problem.

**Comment by huard** [comment]
I think this would fix #13

**Comment by tlogan2000** [comment]
@huard @Zeitsperre  made a few changes adds to the workflow notebook.  I think we should be good to go ...

**Comment by Zeitsperre** [comment]
@huard I still have a few things to add to `usage.py` but feel free to review.

**Comment by huard** [comment]
My feeling is that the notebook is too long, and should start with a simple computation of a climate indicator, highlighting the fact that all the netcdf attributes are there, that units are handled, etc. 

I would create another notebook for subsetting utilities., and another about optimizations. 

That being said, I'm totally OK with this PR as is and we can fine-tune later, including using a web accessible input file.

File: docs/usage.rst [hunk 1]
```diff
@@ -1,13 +1,95 @@
...
+
+Slicing and subsetting with xarray
+----------------------------------
+
+
+Using the xclim.icclim module
+-----------------------------
+`ICCLIM <https://github.com/cerfacs-globc/icclim>`_ is a software platform / python library for performing climate indice calculations. It includes roughly 50 types of indicators as defined by the `European Climate Assessment & Dataset project <https://www.ecad.eu/>`_.
+
+For those familiar with ICCLIM, xclim has created one-to-one mappings of their indices. Effectively, these are simply indice calculations methods that have been renamed to be consistent with ICCLIM's terminology.
```
Comment (huard) [inline docs/usage.rst]: not all indices, and not a perfect mapping.

**Pull request merged**